questions/questions.ts

```diff
@@ -354,11 +354,8 @@ export const askProQuestions = async (program: QuestionArgs) => {
     // default to use LlamaParse if using LlamaCloud
     program.useLlamaParse = true;
   } else {
-    // Extractor template doesn't support LlamaParse and LlamaCloud right now (cannot use asyncio loop in Reflex)
-    if (
-      program.useLlamaParse === undefined &&
-      program.template !== "extractor"
-    ) {
+    // Reflex template doesn't support LlamaParse and LlamaCloud right now (cannot use asyncio loop in Reflex)
```

Comment:
@leehuwuj please add a ticket for this if we continue using more reflex with this PR

Reply:
Sure, I think it's worth logging a ticket for it: https://github.com/run-llama/create-llama/issues/468